tests/utils/hasOwnPop.spec.js

```diff
@@ -0,0 +1,27 @@
+import hasOwnProp from '../../utils/hasOwnProp'
```

Comment:
There is a typo in the test file name.

Reply:
5616f7bd30909b91d72058f49a2ee555da4e8c6e